src/themes/custom_theme/custom_theme.css

```diff
@@ -0,0 +1,18 @@
+.custom_theme {
+	display: inline-block;
+}
+.custom_theme input {
+    width: 100px;
+}
+#properties {
```

Comment:
@4p00rv Use .custom_theme #properties 
Its better to use .properties than #properties because properties is very common name and could be used anybody in other places of webtrader